### PR TITLE
Update go.yml

### PR DESCRIPTION
**Pull request by liberodark**

### CODE REVIEW

**Comment by liberodark** [comment]
Have add the libs for compil on ubuntu

**Comment by liberodark** [comment]
Is just for test with your CI

**Comment by essial** [comment]
Thank you so much!

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       id: go
```
Comment (hansomejason) [inline .github/workflows/go.yml]: APIcloud Attach files by dragging & dropping, selecting or pasting them.

**Pull request merged**